odd_collector/adapters/databricks/mappers/types.py

```diff
@@ -0,0 +1,23 @@
+# https://learn.microsoft.com/en-us/azure/databricks/sql/language-manual/sql-ref-datatypes
+
+from odd_models.models import Type
+
+TYPES_SQL_TO_ODD: dict[str, Type] = {
+    "float": Type.TYPE_NUMBER,
+    "struct": Type.TYPE_STRUCT,
```

Comment:
Check all complex types response, if you can retrieve subtypes write logic to map them